### DISCUSSION TITLE
Weight-based dosing with a different body size metric (lean body weight, FFM, BSA, etc)

### DISCUSSION THREAD

**Discussion by jgg1822** [Open-Systems-Pharmacology/Forum#1003] [Questions & Problems]
Hi all,

I'm wondering if there is an easy way to implement weight-based dosing with a body size metric other than total body weight (BW in Mobi)? That is, using something like fat-free mass, lean body weight, or BSA with mg/kg dosing?

Thanks!

**Comment by msevestre** [comment]
@jgg1822 
You can define your dose as mg/ SA 
![image](https://user-images.githubusercontent.com/1041237/138873461-ec62976f-22dc-4df1-bbcb-902d088a2d11.png)

Isn't it what you are looking for?

**Comment by jgg1822** [comment]
Thanks for the quick response! This makes sense for BSA dosing. Is there a way to do this for lean body weight dosing, etc? So instead of calculating the total dose (in mg, for example) multiplying be total body weight for mg/kg dosing, you multiply by another body size measure (in kg) instead? My guess is I will need to go into Mobi and define a new body size measure, then change the equation for total dose from BW to this new measure.
